services/app-api/serverless.yml

```diff
@@ -14,7 +14,8 @@ plugins:
   - serverless-associate-waf
   - serverless-offline
   - serverless-stack-termination-protection
-  - serverless-enable-api-logs
+  - serverless-idempotency-helper
```

Comment:
This is how you enable the serverless-idempotency-helper.  No config required or expected.